package.json

```diff
@@ -228,13 +228,13 @@
     "flow-bin": "^0.128.0",
     "html-tag-names": "^1.1.2",
     "husky": "^3.0.9",
-    "jest": "^25.4.0",
-    "jest-environment-jsdom": "^25.4.0",
+    "jest": "^26.4.2",
+    "jest-environment-jsdom": "^26.3.0",
     "jest-in-case": "^1.0.2",
-    "jest-junit": "^10.0.0",
+    "jest-junit": "^11.1.0",
     "jest-serializer-html": "^7.0.0",
-    "jest-watch-typeahead": "^0.5.0",
-    "jsdom": "^16.2.2",
+    "jest-watch-typeahead": "^0.6.1",
```

Comment:
I've wanted to finally leverage [improved exact typeahead matching](https://github.com/jest-community/jest-watch-typeahead/pull/39) hence this update here